lib/l10n-es_ES.js

```diff
@@ -21,7 +21,9 @@ exports.messages = {
 ,   "headers.h2-status.no-h2":  "No status+date h2 element."
 ,   "headers.h2-status.bad-h2": "Incorrect status h2 header."
     // headers/h2-toc
-,   'headers.h2-toc.missing':  'There is no table of contents after the status section, labeled with a <code>&lt;h2&gt;</code> element with content <em>&ldquo;Table of Contents&rdquo;</em>.'
+,   'headers.h2-toc.not-found':  'There is no table of contents inside a navigation element (<code>&lt;nav id="toc"&gt; &hellip; &lt;h2&gt;Table of Contents&lt;/h2&gt; &hellip; &lt;/nav&gt;</code>).'
+,   'headers.h2-toc.too-many':   'Found more than one navigation elements; possibly containing the table of contents. There should be just one <code>&lt;nav id="toc"&gt;</code>.'
+,   'headers.h2-toc.not-html5':  'Support for old HTML doctypes is deprecated and will be abandoned soon; please upgrade to HTML5. The expected navigation element is: <code>&lt;nav id="toc"&gt;</code>.'
```

Comment:
That message refers to the rule `headers.h2-toc`. That's what the user currently sees on the interface. I'm afraid he will get confused by these messages referring to the navigation.